### PR TITLE
README.md updated - slackarchive removed

### PR DESCRIPTION
**Pull request by idvoretskyi**
Fixes https://github.com/kubernetes/community/issues/2605

cc @dholbach @castrojo @parispittman

### CODE REVIEW

File: communication/README.md [hunk 1]
```diff
@@ -25,9 +25,6 @@ and meetings devoted to Kubernetes.
 * [Slack] - sign up  
 
 Real time discussion at kubernetes.slack.io:   
-Discussions on most channels are archived at [kubernetes.slackarchive.io].
```
Comment (nikhita) [inline communication/README.md]: @idvoretskyi can we also remove the `Real time discussion at kubernetes.slack.io:` line above?

**Comment by dholbach** [comment]
There's also a mention in `contributors/guide/issue-triage.md` and `sig-governance.md`.

File: communication/README.md (hunk 1 above)
Comment (dholbach) [reply communication/README.md]: https://github.com/kubernetes/community/blame/5b4146f10df50b6255a71144efd6b87bbeea8eb8/communication/README.md#L99 too.

**Comment by dholbach** [comment]
:+1:

**Comment by idvoretskyi** [comment]
@dholbach @nikhita comments addressed, thanks.

**Comment by nikhita** [comment]
@idvoretskyi can you squash the commits? lgtm after that :smile:

**Pull request merged**